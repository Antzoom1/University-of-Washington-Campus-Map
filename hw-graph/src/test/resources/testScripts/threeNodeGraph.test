# Tests adding three nodes to the graph

# Create a graph
CreateGraph threeNodesGraph

# Add three nodes
AddNode threeNodesGraph 1
AddNode threeNodesGraph 2
AddNode threeNodesGraph 3

# Print the nodes in the graph, the outgoing edges from 1,
# 2 and 3
ListNodes threeNodesGraph
ListChildren threeNodesGraph 1
ListChildren threeNodesGraph 2
ListChildren threeNodesGraph 3

