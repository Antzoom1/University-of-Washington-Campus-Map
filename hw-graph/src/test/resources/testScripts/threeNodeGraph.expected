# Tests adding three nodes to the graph

# Create a graph
created graph threeNodesGraph

# Add three nodes
added node 1 to threeNodesGraph
added node 2 to threeNodesGraph
added node 3 to threeNodesGraph

# Print the nodes in the graph, the outgoing edges from 1,
# 2 and 3
threeNodesGraph contains: 1 2 3
the children of 1 in threeNodesGraph are:
the children of 2 in threeNodesGraph are:
the children of 3 in threeNodesGraph are: